content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -176,9 +179,16 @@ network of the consumers.
 ## Control plane models
 
 An Istio mesh uses the {{< gloss >}}control plane{{< /gloss >}} to configure all
-communication between workload instances within the mesh. You can replicate the
-control plane, and workload instances connect to any control plane instance to
-get their configuration.
+communication between workload instances within the mesh. At one extreme, you can choose
+to deploy one control plane per cluster in the mesh, while at the other extreme,
+you can host one single shared control plane for all clusters in the mesh.
+
+_Note: Each control plane will only honor configurations (such as Gateways, VirtualServices, etc.)
+that are present in its local Kubernetes cluster. If you wish to apply a configuration to all workloads in the mesh
```

Comment:
This is no longer true with the external control plane stuff, there is a concept of a config cluster where an external control plane can pull its config. So we should maybe add a parenthetical that mentions that external control planes pull their config from a single config cluster.

Reply:
+1, local k8s cluster is really our new 'config cluster' now, as it doesn't have to be local.